modules/ROOT/pages/faq.adoc

```diff
@@ -27,12 +27,20 @@ However, {variant-name} is specifically focused on workstation and desktop use c
 
 == About using {variant-name}
 
+[[how-can-i-play-more-videos-in-firefox-like-youtebe]]
 === How can I play more videos in Firefox, like YouTube?
 
 Firefox is included in the OS image for now (see https://github.com/fedora-silverblue/issue-tracker/issues/288[issue#288] for progress on moving to Flatpak by default).
 Until that changes, getting it to play videos works the same way as it does for the regular {variant-classic}: find a package with the needed codecs, and install it.
 The one difference is that you use `rpm-ostree install` instead of `dnf install`.
-An alternative solution is to install https://flathub.org/apps/details/org.mozilla.firefox[Firefox from Flathub].
+
+An alternative solution is to install https://flathub.org/apps/details/org.mozilla.firefox[Firefox from Flathub] by running these commands:
```

Comment:
We recommend hiding the default browser instead as that does not changes the image and the result is the same: https://docs.fedoraproject.org/en-US/fedora-silverblue/tips-and-tricks/#_hiding_the_default_browser_firefox

We also have a note about Flathub somewhere so we should link to that and to the Firefox page on Flathub instead of command lines ideally.